docs/reference/query-dsl/queries/function-score-query.asciidoc

```diff
@@ -193,6 +194,7 @@ There are a number of options for the `field_value_factor` function:
 |`modifier` |Modifier to apply to the field value, can be one of: `none`, `log`,
  `log1p`, `log2p`, `ln`, `ln1p`, `ln2p`, `square`, `sqrt`, or `reciprocal`.
  Defaults to `none`.
+|`default` |Default value used if the document doesn't have that field.
```

Comment:
Maybe we should make explicit that the `modifier` would still be applied in that case?